manuscript/10-Arrays.md

```diff
@@ -738,7 +738,7 @@ Three typed arrays are created from the original `ints` array in this example. T
 
 ## Summary
 
-ECMAScript 6 continues the work of ECMAScript 5 by making arrays more useful. There are two more ways to create arrays: the `Array.of()` and `Array.from()` methods. The `Array.from()` method can also convert iterables and array-like objects into arrays. Both methods are inherited by derived array classes and use the `Symbol.species` property to determine what type of value should be returned (other inherited methods also use `Symbol.species` when returning an array).
+ECMAScript 6 continues the work of ECMAScript 5 by making arrays more useful. There are two more ways to create arrays: the `Array.of()` and `Array.from()` methods. The `Array.from()` method can also convert iterables and array-like objects into arrays. Both methods are inherited by derived array classes. They **do not** use the `Symbol.species` property to determine what type of value should be returned but other inherited methods use it when returning an array).
```

Comment:
This fact is mentioned when `Array.of` and `Array.from` are discussed separately in this chapter.

Reply:
Thanks. Stylistically I can't use bold in this way, but I'll fix in another commit.